plugins/weaponblocking/commands.py

```diff
@@ -0,0 +1,25 @@
+from core import DCSServerBot, Plugin
+from .listener import WeaponBlockingListener
+
+
+class WeaponBlocking(Plugin):
+    """
```

Comment:
This is left in from the example plugin.
Removing it causes formatting errors.
Is this class even needed? If so what should be in this comment's place?